pyiron_workflow/create.py

```diff
@@ -6,13 +6,22 @@
 
 from concurrent.futures import ProcessPoolExecutor, ThreadPoolExecutor
 
-from executorlib import Executor as ExecutorlibExecutor
+import executorlib
 from pyiron_snippets.dotdict import DotDict
 
 from pyiron_workflow.executors import CloudpickleProcessPoolExecutor
 from pyiron_workflow.nodes.function import as_function_node, function_node
 
 
+class ExecutorlibExecutor:
+    def __init__(self):
+        self.SlurmJobExecutor = executorlib.SlurmJobExecutor
+        self.SingleNodeExecutor = executorlib.SingleNodeExecutor
+        self.FluxJobExecutor = executorlib.FluxJobExecutor
+        self.FluxClusterExecutor = executorlib.FluxClusterExecutor
+        self.SlurmClusterExecutor = executorlib.SlurmClusterExecutor
```

Comment:
```suggestion
    SlurmJobExecutor = executorlib.SlurmJobExecutor
    SingleNodeExecutor = executorlib.SingleNodeExecutor
    FluxJobExecutor = executorlib.FluxJobExecutor
    FluxClusterExecutor = executorlib.FluxClusterExecutor
    SlurmClusterExecutor = executorlib.SlurmClusterExecutor
```

I guess these can just be class attributes, no need to wait for the instance